pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -25,6 +25,12 @@
 from pandas import (isna, to_datetime, Timestamp, Series, DataFrame,
                     Index, DatetimeIndex, NaT, date_range, compat)
 
+if PY3:
+    from datetime import timezone
```

Comment:
I think we have a fixture for this

Reply:
We have a fixture for `timezone.utc`, but I am testing parsing a custom timezone.